rpc-server/Cargo.toml

```diff
@@ -60,6 +61,8 @@ near-vm-runner = "0.17.0"
 near-vm-logic = "0.17.0"
 near-vm-errors = "0.17.0"
 
+near-lake-framework = { path = "../../near-lake-framework-rs" }
```

Comment:
Marking it here to make sure we update it with the release of NEAR Lake Framework

Reply:
related from this pr https://github.com/near/near-lake-framework-rs/pull/86